ssl/ssl_lib.c

```diff
@@ -2231,7 +2231,7 @@ int SSL_export_keying_material(SSL *s, unsigned char *out, size_t olen,
                                const unsigned char *p, size_t plen,
                                int use_context)
 {
-    if (s->version < TLS1_VERSION)
+    if (s->version < TLS1_VERSION && s->version != DTLS1_BAD_VER)
```

Comment:
Because for SSL3 it would end up calling `ssl_undefined_function()`, I suppose.
It's only defined for newer protocols. But that does include DTLS1_BAD_VER.